crates/pop-cli/README.md

```diff
@@ -0,0 +1,258 @@
+```
+
+> :information_source: We plan to automate this in the future.
+
+Deploy and instantiate the smart contract:
+
+```sh
+pop up contract -p ./my_contract --constructor new --args "false" --suri //Alice
+```
+
+Some of the options available are:
+
+- Specify the contract `constructor `to use, which in this example is `new()`.
+- Specify the argument (`args`) to the constructor, which in this example is `false`.
+- Specify the account uploading and instantiating the contract with `--suri`, which in this example is the default
+  development account of `//Alice`.
+  For other accounts, the actual secret key must be provided e.g. an 0x prefixed 64 bit hex string, or the seed phrase.
+
+> :warning: **Use only for development**: Use a safer method of signing here before using this feature with production
+> projects. We will be looking to provide alternative solutions in the future!
+
+- You also can specify the url of your node with `--url ws://your-endpoint`, by default it is
+  using `ws://localhost:9944`.
+
+For more information about the options,
+check [cargo-contract documentation](https://github.com/paritytech/cargo-contract/blob/master/crates/extrinsics/README.md#instantiate)
+
+Interacting with the Smart Contract:
+
+1. Read-only Operations: For operations that only require reading from the blockchain state. This approach does not
+   require to submit an extrinsic (skip the flag `x/--execute`).
+   Example using the get() message:
+
+```sh
+pop call contract -p ./my_contract --contract $INSTANTIATED_CONTRACT_ADDRESS --message get --suri //Alice
+```
+
+2. State-modifying Operations: For operations that change a storage value, thus altering the blockchain state. Include
+   the `x/--execute`  flag to submit an extrinsic on-chain.
+
+Example executing the `flip()` message:
+
+```sh
+pop call contract -p ./my_contract --contract $INSTANTIATED_CONTRACT_ADDRESS --message flip --suri //Alice -x
+```
+
+### E2E testing
+
+For end-to-end testing you will need to have a Substrate node with `pallet contracts`.
+You do not need to run it in the background since the node is started for each test independently.
+To install the latest version:
+
+```
+cargo install contracts-node --git https://github.com/paritytech/substrate-contracts-node.git
+```
+
+If you want to run any other node with `pallet-contracts` you need to change `CONTRACTS_NODE` environment variable:
+
+```
+export CONTRACTS_NODE="YOUR_CONTRACTS_NODE_PATH"
+```
+
+Run e2e testing on the smart contract:
+
+```sh
+# Run e2e tests for an existing smart contract
+ pop test contract  -p ./my_contract --features e2e-tests
+```
+
+## Building Pop CLI locally
+
+Build the tool locally with all the features:
+
+```sh
+cargo build --all-features
+```
+
+Build the tool only for parachain functionality:
+
+```sh
+cargo build --features parachain
+```
+
+Build the tool only for contracts functionality:
+
+```sh
+cargo build --features contract
+```
+
+## Spawn Network using Zombienet
```

Comment:
Replace above in section "Parachains" so that there is a clear separation in the README between parachain & contract

Reply:
simply the lines:
```shell
pop up parachain -f ./tests/zombienet.toml -p https://github.com/r0gue-io/pop-node
```


> :information_source: Pop CLI will automatically source the necessary polkadot binaries. Currently, these will be built
> if on a non-linux system.